.github/scripts/get_latest_runtime_image_tag.py

```diff
@@ -25,7 +26,10 @@ def get_latest_tag(repository: str) -> str:
         except ValueError:
             continue
     if valid_tags:
+        print(valid_tags)
```

Comment:
Can we remove this? 
```suggestion
```

Reply:
I thought this line was for easy debugging?